sdk/sdk-view/src/test/java/mil/emp3/api/utils/ComparisonUtils.java

```diff
@@ -0,0 +1,171 @@
+        assertEquals(c1.getBlue(), c2.getBlue());
+        assertEquals(c1.getGreen(), c2.getGreen());
+    }
+
+    private static void compareFillStyle(final IGeoFillStyle f1, final IGeoFillStyle f2) {
+        assertEquals(f1.getDescription(), f2.getDescription());
+        compareGeoColor(f1.getFillColor(), f2.getFillColor());
+        assertEquals(f1.getFillPattern(), f2.getFillPattern());
+    }
+
+    private static void compareStrokeStyle(final IGeoStrokeStyle s1, final IGeoStrokeStyle s2) {
+        assertEquals(s1.getStipplingFactor(), s2.getStipplingFactor());
+        assertEquals(s1.getStipplingPattern(), s2.getStipplingPattern());
+        compareGeoColor(s1.getStrokeColor(), s2.getStrokeColor());
+        assertEquals(s1.getStrokeWidth(), s2.getStrokeWidth(), Epsilon);
+    }
+
+    private static void compareIconStyle(final IGeoIconStyle i1, final IGeoIconStyle i2) {
+        assertEquals(i1.getOffSetX(), i2.getOffSetX(), Epsilon);
+        assertEquals(i1.getOffSetY(), i2.getOffSetY(), Epsilon);
+        assertEquals(i1.getSize(), i2.getSize(), Epsilon);
+    }
+
+    private static void compareLabelStyle(final IGeoLabelStyle l1, final IGeoLabelStyle l2) {
+        compareGeoColor(l1.getColor(), l2.getColor());
+        assertEquals(l1.getFontFamily(), l2.getFontFamily());
+        assertEquals(l1.getJustification(), l2.getJustification());
+        compareGeoColor(l1.getOutlineColor(), l2.getOutlineColor());
+        assertEquals(l1.getSize(), l2.getSize(), Epsilon);
+        assertEquals(l1.getTypeface(), l2.getTypeface());
+    }
+
+    private static void compareGeoPoint(final IGeoPoint p1, final IGeoPoint p2) {
+        assertEquals(p1.getTimeStamp(), p2.getTimeStamp());
+        assertEquals(p1.getTimeSpans(), p2.getTimeSpans());
+        assertEquals(p1.getTessellate(), p2.getTessellate());
+        compareStrokeStyle(p1.getStrokeStyle(), p2.getStrokeStyle());
+        assertEquals(p1.getReadOnly(), p2.getReadOnly());
+        assertEquals(p1.getProperties(), p2.getProperties());
+        compareGeoPositionArray(p1.getPositions(), p2.getPositions());
+        assertEquals(p1.getPathType(), p2.getPathType());
+        assertEquals(p1.getName(), p2.getName());
+        compareLabelStyle(p1.getLabelStyle(), p2.getLabelStyle());
+        compareFillStyle(p1.getFillStyle(), p2.getFillStyle());
+        assertEquals(p1.getExtrude(), p2.getExtrude());
+        assertEquals(p1.getDescription(), p2.getDescription());
+        assertEquals(p1.getDataProviderId(), p2.getDataProviderId());
+        assertEquals(p1.getChildren(), p2.getChildren());
+        assertEquals(p1.getBuffer(), p2.getBuffer(), Epsilon);
+        assertEquals(p1.getAzimuth(), p2.getAzimuth(), Epsilon);
+        assertEquals(p1.getAltitudeMode(), p2.getAltitudeMode());
+        assertEquals(p1.getIconURI(), p2.getIconURI());
+        compareIconStyle(p1.getIconStyle(), p2.getIconStyle());
+    }
+
+    public  static void validatePoint(final Point point,
```

Comment:
there is an extra space between public and static, please remove.  Also, should stick to the same naming convention, either all of them should be validate<ClassName> or all of them should be compare<ClassName>

Reply:
so the convention I was planning to follow is comparison is between two objects, validate is between an object and a list of values.  Is that all right or should I just make them all compare?

Reply:
that works, I didn't think of that difference.

Reply:
space should still be removed though